scripts/run-integration-tests.sh

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+DIR=$(cd "$(dirname "$0")"; pwd)
+source $DIR/up-test-cluster.sh
+source $DIR/down-test-cluster.sh
+
+OS=$(go env GOOS)
+ARCH=amd64
+TEST_ID=$RANDOM
+CLUSTER_NAME=test-cluster-$TEST_ID
+BASE_DIR=$(dirname $0)
+TEST_DIR=/tmp/cni-test
+REPORT_DIR=${TEST_DIR}/report
+REGION=${AWS_REGION:-us-west-2}
+K8S_VERSION=${K8S_VERSION:-1.14.1}
+PROVISION=${PROVISION:-true}
+DEPROVISION=${DEPROVISION:-true}
+BUILD=${BUILD:-true}
+
+echo "Testing in region: $REGION"
+
+TESTER_DOWNLOAD_URL=https://github.com/aws/aws-k8s-tester/releases/download/v0.4.3/aws-k8s-tester-v0.4.3-$OS-$ARCH
+TESTER_PATH=$TEST_DIR/aws-k8s-tester
```

Comment:
Recommend *not* putting the `aws-k8s-tester` binary in `$TEST_DIR` because you `rm -rf $TEST_DIR` at the end of each test run...

Reply:
Removed `rm -rf $TEST_DIR` in favor of leaving behind `/tmp/$TEST_DIR` for examination